tests/testthat/test-merge_sce_list.R

```diff
@@ -417,39 +498,61 @@ test_that("merging SCEs with different altExp features works as expected, with a
     preserve_rowdata_cols = c("gene_names")
   )
 
-  merged_altexp <- altExp(merged_sce)
+  expect_true(altExpNames(merged_sce) == "adt")
```

Comment:
I think I would also add a few more tests here that the altExp matrices are correct: Are the values from `sce4` all NA? Are the values from other sces the same as what went in?
Maybe also check that the dimensions are what is expected?